src/components/pages/room.yml

```diff
@@ -44,3 +44,18 @@ room:
   pendingInvitation:
     en: Pending invitation
     de: Ausstehende Einladung
+  messageBoardSectionHeadline:
+    en: Message board
+    de: Message board
+  postMessageButtonText:
+    en: Post message
+    de: Nachricht abschicken
+  postMessageCheckboxText:
+    en: Send email to members
+    de: E-Mail an Mitglieder schicken
+  postMessageCheckboxInfo:
+    en: An email will be immediately sent to all room members, notifying them that there is new information on the room message board
+    de: Eine E-Mail wird sofort an alle Raummitglieder geschickt, notifying them that there is new information on the room message board
+  messageEmailNotificationIconTooltip:
+    en: An email was sent to all room members when this message was posted
+    de: An email was sent to all room members when this message was posted
```

Comment:
Eine E-Mail wurde an alle Raum-Mitglieder geschickt, als diese Nachricht eingestellt wurde

Reply:
Changed this too, thanks for all the translating 🙏